user/util.go

```diff
@@ -2,11 +2,29 @@ package user
 
 import (
 	"crypto/ecdsa"
+	"fmt"
 
 	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
 )
 
 // IDFromKey forms the ID using script hash calculated for the given key.
 func IDFromKey(id *ID, key ecdsa.PublicKey) {
```

Comment:
I see you're making it consistent with this one at the same time. Do we want to change it, @cthulhu-rider? Or maybe it's not worth the trouble for now?

Reply:
BTW, is it used anywhere at all after this patch?

Reply:
Yes, it is using in some test and in bearer token https://github.com/nspcc-dev/neofs-sdk-go/blob/dbbb22ca28cd1f60d507c7cec35985dc5f487274/bearer/bearer.go#L359

Reply:
it makes no difference to me to change it or not, but if I do it, then before the release

i guess this function accepting particular key type shouldn't be needed anymore since we are moving to generic `Signer/PublicKey` pairs

Reply:
Function `IDFromKey` changed to use `[]byte` instead of ecdsa.PublicKey. Problem gone